src/llfs/storage_file_builder.test.cpp

```diff
@@ -65,14 +156,31 @@ class StorageFileBuilderTest : public ::testing::Test
 //  order.
 //
 
-TEST_F(StorageFileBuilderTest, NoConfigs)
+TEST_F(StorageFileBuilderTest, NoConfigs_Mock)
 {
   ::testing::StrictMock<llfs::RawBlockFileMock> file_mock;
 
   llfs::StorageFileBuilder builder{file_mock, /*base_offset=*/0};
 }
 
-TEST_F(StorageFileBuilderTest, PageDeviceConfig_NoFlush)
+TEST_F(StorageFileBuilderTest, NoConfigs)
+{
+  llfs::StatusOr<llfs::ScopedIoRing> ioring =
+      llfs::ScopedIoRing::make_new(llfs::MaxQueueDepth{1024}, llfs::ThreadPoolSize{1});
+
+  ASSERT_TRUE(ioring.ok()) << BATT_INSPECT(ioring.status());
+
+  const char* const test_file_name = "/dev/nvme8n1";
```

Comment:
Thinking the best way to achieve this will be using loopback devices and flat files. Can use a docker file to map names of the generic test files/block files to real locations on the users machine. Using flat file in docker with no linking for the time being.